### ISSUE TITLE
A possible solution to the deprecation of require.extensions

### ISSUE THREAD

**Issue by CamiloMM** [olalonde/better-require#8]
Since [require.extensions is deprecated](http://nodejs.org/api/all.html#all_require_extensions), you could add the option of:

``` js
var betterRequire = require('better-require')('json xml yaml');
```

You then could even do this (if it doesn't make you feel bad inside):

``` js
var require = require('better-require')('json xml yaml');
```

**Comment by olalonde** [comment]
@CamiloMM Good idea. I don't have time to maintain this project anymore unfortunately. Would you be interested to take over?

**Comment by CamiloMM** [comment]
@olalonde Sorry, I'm in the same "have to maintain less stuff" vibe :/

If only we had unlimited time in our hands, eh? :)
